js/insult.js

```diff
@@ -23,6 +23,8 @@
 		'Wang',
 		'Cum',
 		'Whore',
+		'Anus',
```

Comment:
Hey Pete,

We went back and forth on whether to include anus, but we're gonna stick with the single-syllable cuss pattern. We do, after all, have "ass" and "butt". Love the rest though. Remove this one and we can merge it in.